src/lib/utils.ts

```diff
@@ -83,7 +83,10 @@ export function mapRpcPubKeyToProto(
       secp256k1: pubkey.data,
     };
   } else {
-    throw new Error(`Unknown validator pubkey type: ${pubkey.algorithm}`);
+    throw new Error(
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      `Unknown validator pubkey type: ${(pubkey as any).algorithm}`
```

Comment:
@webmaster128 Maybe these types should extend a base pubkey which has an algorithm string property?

Reply:
Since `pubkey` is of type `never` in this code branch and it is only a safety exception, I'd not over-engineer it. Let's just use
```js
throw new Error(`Unknown validator pubkey type`);
````